### PR TITLE
ZX-169-New: Fixes for code review comments in zm-x-web branch

### PR DESCRIPTION
**Pull request by craigdanj**
Profile image issue resolved for account photo in header.
Width and height of profile image fetched has been adjusted.

### CODE REVIEW

File: src/schema/schema.graphql [hunk 1]
```diff
@@ -272,6 +272,9 @@ type AuthResponse {
 	skin: [Skin]
 }
 
+type ProfileImageChangeResponse {
+	itemId: Int
+}
```
Comment (silentsakky) [inline src/schema/schema.graphql]: space after }

File: src/schema/schema.graphql [hunk 2]
```diff
@@ -272,6 +272,9 @@ type AuthResponse {
 	skin: [Skin]
 }
 
+type ProfileImageChangeResponse {
+	itemId: Int
```
Comment (silentsakky) [inline src/schema/schema.graphql]: ID

File: src/schema/schema.graphql (hunk 1 above)
Comment (craigdanj) [reply src/schema/schema.graphql]: fixed

File: src/schema/schema.graphql (hunk 2 above)
Comment (craigdanj) [reply src/schema/schema.graphql]: fixed

File: src/batch-client/index.ts [hunk 3]
```diff
@@ -271,7 +271,8 @@ export class ZimbraBatchClient {
 			body: {
 				...denormalize(CalendarItemCreateModifyRequest)(appointment)
 			},
-			accountName: accountName
+			accountName: accountName,
```
Comment (lavegupta) [inline src/batch-client/index.ts]:
```
accountName,
singleRequest: true
```

File: src/batch-client/index.ts [hunk 4]
```diff
@@ -283,7 +284,8 @@ export class ZimbraBatchClient {
...
```
Comment (lavegupta) [inline src/batch-client/index.ts]:
```
accountName,
singleRequest: true
```

File: src/batch-client/index.ts [hunk 5]
```diff
@@ -632,7 +636,8 @@ export class ZimbraBatchClient {
...
```
Comment (lavegupta) [inline src/batch-client/index.ts]: same as above

File: src/batch-client/index.ts (hunk 3 above)
Comment (craigdanj) [reply src/batch-client/index.ts]: fixed

File: src/batch-client/index.ts (hunk 4 above)
Comment (craigdanj) [reply src/batch-client/index.ts]: fixed

File: src/batch-client/index.ts (hunk 5 above)
Comment (craigdanj) [reply src/batch-client/index.ts]: fixed

**Pull request merged**